host/src/gatt.rs

```diff
@@ -227,7 +239,10 @@ fn process_accept<'stack>(
     connection: &Connection<'stack>,
     server: &dyn DynamicAttributeServer,
 ) -> Result<Reply<'stack>, Error> {
-    let att = unwrap!(AttReq::decode(pdu.as_ref()));
+    let att = unwrap!(Att::decode(pdu.as_ref()));
+    let Att::Client(att) = att else {
+        unreachable!();
+    };
```

Comment:
Same as earlier.

Reply:
See above - I'll put something in the unreachable!()